.travis.yml

```diff
@@ -60,37 +75,27 @@ matrix:
             - gcc-7
             - g++-7
       env: DEVITO_ARCH=gcc-7 DEVITO_OPENMP=0 DEVITO_BACKEND=ops YC_CXX=g++-7 INSTALL_TYPE=conda RUN_EXAMPLES=False MPI_INSTALL=0
-  allow_failures:
```

Comment:
what was this doing before? I have absolutely no idea

Reply:
With was allowing failure on [ython 2.7 which is not tested pr supported so was useless